### ISSUE TITLE
Installation fails on Fedora

### ISSUE THREAD

**Issue by Enchufa2** [nlmixr2/rxode2#350]
Package build fails since v2.0.9 (see [build list](https://copr.fedorainfracloud.org/coprs/iucar/cran/package/R-CRAN-rxode2/)). First, I see:

```
> .in <- suppressWarnings(readLines("src/Makevars.in"))
> .in <- gsub("@ARMA@", file.path(find.package("RcppArmadillo"),"include"), .in)
> .in <- gsub("@BH@", file.path(find.package("BH"),"include"), .in)
> .in <- gsub("@RCPP@", file.path(find.package("Rcpp"),"include"), .in)
> .in <- gsub("@EG@", file.path(find.package("RcppEigen"),"include"), .in)
Error in find.package("RcppEigen") : 
  there is no package called 'RcppEigen'
Calls: gsub -> file.path -> find.package
Execution halted
--------[begin src/Makevars]--------
--------[end src/Makevars]--------
--------[begin R/RxODE_md5.R]--------
rxode2.md5 <- "61791724322c7b105f379eeb433c3f82"

--------[end R/RxODE_md5.R]--------
cat: src/Makevars: No such file or directory
```

Not sure why, because RcppEigen is not declared as a dependency. And then,

```
etTran.cpp: In function 'Rcpp::List etTrans(Rcpp::List, const Rcpp::RObject&, bool, bool, bool, bool, Rcpp::Nullable<Rcpp::Vector<10, Rcpp::PreserveStorage> >, Rcpp::CharacterVector)':
etTran.cpp:891:20: error: 'max2' was not declared in this scope
  891 |         maxShift = max2(maxShift, lastTime-ctime);
      |                    ^~~~
etTran.cpp:974:13: error: 'max2' was not declared in this scope
  974 |     mxCmt = max2(cmt,mxCmt);
      |             ^~~~
etTran.cpp:1878:64: warning: unused variable 'cur1' [-Wunused-variable]
 1878 |                                                         double cur1 = nvTmp[idx1];
      |                                                                ^~~~
make: *** [/usr/lib64/R/etc/Makeconf:177: etTran.o] Error 1
ERROR: compilation failed for package 'rxode2'
```

See the complete build log [here](https://download.copr.fedorainfracloud.org/results/iucar/cran/fedora-37-x86_64/04999460-R-CRAN-rxode2/builder-live.log.gz).

**Comment by mattfidler** [comment]
Hi @Enchufa2 

I took out the RcppEigen, but I believe your sed is taking out the required -D_isrxode2_ which defines `max2`

**Comment by mattfidler** [comment]
You can see that the compile flags do not include the -D_isrxode2_ in the definitions.  It could be because the Makevars isn't writen correctly since it errored at the `RcppEigen` stage

**Comment by mattfidler** [comment]
It may not be the sed.

**Comment by Enchufa2** [comment]
I tried with the fix above, and now there's another error message about StanHeaders, see https://download.copr.fedorainfracloud.org/results/iucar/cran/fedora-37-x86_64/05000235-R-CRAN-rxode2/builder-live.log.gz.

**Comment by mattfidler** [comment]
`StanHeaders` are not needed either.

**Comment by mattfidler** [comment]
I removed them from the Makevars and from the configure script.

**Comment by mattfidler** [comment]
I think for the CRAN and other builders there were seen because they were installed with the dependencies.

**Comment by Enchufa2** [comment]
I confirm this fixes the issue: https://download.copr.fedorainfracloud.org/results/iucar/cran/fedora-37-x86_64/05000555-R-CRAN-rxode2/builder-live.log

**Comment by mattfidler** [comment]
I asked for `rxode2` `2.0.11` to be recalled from CRAN submission yesterday to try to put this in, but they haven't responded.  

I guess I will resubmit with the same version number and see if they respond that way.

**Comment by mattfidler** [comment]
The new version on CRAN should fix this ie `rxode2` `2.0.11`
